### PR TITLE
Fix tasks get_schema

### PR DESCRIPTION
**Pull request by Riul1999**
# Summary

Refactor the **Task** classes to remove the _get_schema_ method and declare the _DESCRIPTION_ attribute.

## Type of change

- Refactoring.

## Changes

- Remove the _get_schema_ class method, the method was not used and a **Task** is not a Configurable Component, so it should not have a _get_schema_ method.
- Add _DESCRIPTION_ attribute to concrete **Task** classes, in order to pass it to the **ComponentRegistry** class.

### CODE REVIEW

File: DashAI/back/tasks/text_classification_task.py [hunk 1]
```diff
@@ -10,6 +10,8 @@
 class TextClassificationTask(BaseTask):
     """Base class for text classifitacion task."""
 
+    DESCRIPTION: str = "Classify data with text features by categories."
```
Comment (pbadillatorrealba) [inline DashAI/back/tasks/text_classification_task.py]:
```suggestion
    DESCRIPTION: str = "Text classification is an essential Natural Language Processing (NLP) task that involves automatically assigning pre-defined categories or labels to text documents based on their content. It serves as the foundation for applications like sentiment analysis, spam filtering, topic classification, and document categorization."
```

File: DashAI/back/tasks/translation_task.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 class TranslationTask(BaseTask):
     """Base class for translation tasks."""
 
+    DESCRIPTION: str = "Translate text training a AI model."
```
Comment (pbadillatorrealba) [inline DashAI/back/tasks/translation_task.py]:
```suggestion
    DESCRIPTION: str = "The translation task is natural language processing (NLP) task that  involves converting text or speech from one language into another language while preserving the meaning and context."
```

File: DashAI/back/tasks/tabular_classification_task.py [hunk 3]
```diff
@@ -9,6 +9,7 @@ class TabularClassificationTask(BaseTask):
     Here you can change the methods provided by class Task.
     """
 
+    DESCRIPTION: str = "Classify data with Tabular features by categories."
```
Comment (pbadillatorrealba) [inline DashAI/back/tasks/tabular_classification_task.py]:
```suggestion
    DESCRIPTION: str = "Tabular classification in machine learning involves predicting categorical labels for structured data organized in tabular form (rows and columns). Models are trained to learn patterns and relationships in the data, enabling accurate classification of new instances."
```

**Review by pbadillatorrealba — CHANGES_REQUESTED** [review]
Changed the original descriptions to ones generated by chatgpt (1313).

**Review by cristian-tamblay — APPROVED** [review]

**Review by pbadillatorrealba — APPROVED** [review]

**Pull request merged**